lib/generators/boring/rack_mini_profiler/install/install_generator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Boring
+  module RackMiniProfiler
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds rack mini profiler to the application'
+
+      def add_rack_mini_profiler_gem
+        say 'Adding rack mini profiler gem', :green
+
+        gem 'rack-mini-profiler', require: false
```

Comment:
Don't we need to bundle install after adding the gem? Is this change verified through testing manually in the rails app?

Reply:
We do need to add bundle install after adding gem.